### PR TITLE
Fix some if-else expressions where indenting makes them unclear

### PR DESCRIPTION
**Pull request by bje-**
This silences G++ warnings about same.

The first hunk adds braces because, by my reading of the code, this is what was intended (and the indenting seems to strongly suggest it, too). The remaining hunks seem to be correct, but need the indenting corrected.

### CODE REVIEW

File: src/invoke.cpp [hunk 1]
```diff
@@ -762,11 +762,12 @@ void fcall_value( lk::invoke_t &cxt )
 		{
 			if ( vv->Read( cxt.arg(1), false ) ){
 			    bool trigger = true;
-			    if (cxt.arg_count() == 3 )
+			    if (cxt.arg_count() == 3 ) {
```
Comment (janinefreeman) [inline src/invoke.cpp:765]: While I agree that this seems to be what is implied by the indentation, adding this curly brace will fundamentally change the behavior of this function. @sjanzou , can you comment whether this looks like the intended behavior, or whether we just need to change indenting for clarity?

Comment (sjanzou) [reply src/invoke.cpp:765]: @janinefreeman, good catch, the proposed change is incorrect (verified in VS2019) 
image
and needs to be 
image


and needs to be:

**Comment by bje-** [comment]
Thanks a lot. It was certainly hard to tell which way the bracing was meant to go. Fixed.

**Comment by bje-** [comment]
Ping?

**Pull request merged**